src/Features/Core/Portable/GenerateMember/GenerateParameterizedMember/AbstractGenerateMethodService.cs

```diff
@@ -35,8 +35,11 @@ internal abstract partial class AbstractGenerateMethodService<TService, TSimpleN
         {
             using (Logger.LogBlock(FunctionId.Refactoring_GenerateMember_GenerateMethod, cancellationToken))
             {
+                var documentOptions = await document.GetCodeGenerationOptionsAsync(fallbackOptions, cancellationToken).ConfigureAwait(false);
+                var allowGenerateInHiddenCode = documentOptions.AllowGenerateInHiddenCode;
```

Comment:
This is odd. The "allow in hidden code" setting is not document-specific (not possible to set in editorconfig file). We read document options here and then only use the one option that is actually not stored in options for the document.

Reply:
I'm not sure what the alternative is? I could read this directly off `fallbackOptions`, which is where it ultimately comes from, but that seems pretty disingenuous.

Reply:
`fallbackOptions` means "if option is not set for the specific document use these options". Since "allow in hidden code" can't be set for the specific document, it seems reasonable to read it directly from fallbackOptions.